package.json

```diff
@@ -2,53 +2,108 @@
   "name": "wd_s",
   "version": "1.0.0",
   "description": "A theme boilerplate for WebDevStudios.",
-  "main": "Gulpfile.js",
-  "dependencies": {
-    "babel-core": "^6.26.3",
-    "babel-preset-env": "^1.7.0",
-    "browser-sync": "^2.26.3",
-    "css-mqpacker": "^6.0.2",
-    "del": "^3.0.0",
-    "eslint": "^4.19.1",
-    "eslint-config-wordpress": "^2.0.0",
-    "glob": "^7.1.2",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^7.0.1",
-    "gulp-cheerio": "^0.6.3",
-    "gulp-concat": "^2.6.1",
-    "gulp-cssnano": "^2.1.3",
-    "gulp-eslint": "^4.0.2",
-    "gulp-imagemin": "^3.4.0",
-    "gulp-notify": "^3.2.0",
-    "gulp-plumber": "^1.2.0",
-    "gulp-postcss": "^7.0.1",
-    "gulp-rename": "^1.2.3",
-    "gulp-replace": "^1.0.0",
-    "gulp-sass": "^3.2.1",
-    "gulp-sass-lint": "^1.4.0",
-    "gulp-sort": "^2.0.0",
-    "gulp-sourcemaps": "^2.6.4",
-    "gulp-svgmin": "^1.2.4",
-    "gulp-svgstore": "^6.1.1",
-    "gulp-uglify": "^3.0.0",
-    "gulp-util": "^3.0.8",
-    "gulp-wp-pot": "^2.3.1",
-    "gulp.spritesmith": "^6.9.0",
-    "natives": "^1.1.6",
-    "sassdoc": "^2.5.1"
-  },
-  "devDependencies": {},
-  "scripts": {
-    "test": "echo \"No test specified\""
+  "author": "WebDevStudios",
+  "license": "GPL-3.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WebDevStudios/wd_s.git"
   },
-  "repository": "git+https://github.com/WebDevStudios/wd_s.git",
   "keywords": [
     "theme"
   ],
-  "author": "WebDevStudios",
-  "license": "GPL-3.0",
   "bugs": {
     "url": "https://github.com/WebDevStudios/wd_s/issues"
   },
-  "homepage": "https://github.com/WebDevStudios/wd_s#readme"
+  "homepage": "https://github.com/WebDevStudios/wd_s#readme",
+  "main": "src/js/app.js",
+  "dependencies": {
+    "slick-carousel": "^1.8.1"
+  },
+  "devDependencies": {
+    "@webdevstudios/js-coding-standards": "^1.0.0-beta1",
+    "@wordpress/eslint-plugin": "^4.0.0",
+    "autoprefixer": "^9.7.4",
+    "browser-sync": "^2.26.7",
+    "cssnano": "^4.1.10",
+    "eslint": "^6.8.0",
+    "imagemin-cli": "^5.1.0",
+    "mkdirp": "^1.0.3",
+    "node-sass": "^4.13.1",
+    "npm-run-all": "^4.1.5",
+    "onchange": "^6.1.0",
+    "postcss-cli": "^7.1.0",
+    "prettier": "^1.19.1",
+    "rimraf": "^3.0.2",
+    "stylelint": "^13.1.0",
+    "stylelint-config-wordpress": "^16.0.0",
+    "stylelint-order": "^4.0.0",
+    "stylelint-scss": "^3.14.2",
+    "svgo": "^1.3.2",
+    "svgstore-cli": "^1.3.1",
+    "uglify-es": "^3.3.9",
+    "wp-pot-cli": "^1.3.0"
+  },
+  "scripts": {
+    "clean": "rimraf dist/{css/*,js/*,images/*,languages/*}",
+    "optimizecss": "postcss dist/css/style.css -c postcss.config.js -o dist/css/style.css",
+    "scss": "node-sass src/scss/style.scss -o dist/css",
+    "gutenberg": "node-sass src/scss/gutenberg-blocks-style.scss -o dist/css",
+    "lint": "eslint src/js || true",
+    "lint-scss": "stylelint src/scss/**/*.scss --syntax scss --config .stylelintrc || true",
```

Comment:
We use `lint:*` style for this elsewhere, but use `lint-*` style here. In https://github.com/WebDevStudios/wd_s/pull/511 I made some of this work, but I leave the `-` style ones around in case it's in documentation, etc.